projects/compass-form/src/lib/compass-form/compass-form.component.ts

```diff
@@ -23,6 +23,7 @@ export class CompassFormComponent<T> {
             width: control.snapshot.width,
             'flex-basis': control.snapshot.width,
             display: control.snapshot.display ? 'block' : 'none',
+            color: control.snapshot.color
```

Comment:
This is setting a the `color` css property for all controls. Is this actually needed? if so can it be done in a place that doesn't impact all controls?

Reply:
Good catch, we do not need that.